DevEdu.API/DevEdu.Business.Tests/TestDataHelpers/MaterialData.cs

```diff
@@ -0,0 +1,93 @@
+﻿using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevEdu.Business.Tests
+{
+    public static class MaterialData
+    {
+        public const int expectedId = 66;
+        public const int tagId = 5;
```

Comment:
it's a bad practice to have such values as consts
it's better to var them in each test method separately